src/continuous/draw/options.ts

```diff
@@ -10,9 +10,19 @@ export interface DrawContinuousOptions extends DrawOptions {
    * @default 100
    */
   slices?: number;
+
+  /**
+   * Border radius of the bar
+   *
+   * 0 will draw a normal rect
+   *
+   * @default 0
+   */
+  roundedRadius?: number;
```

Comment:
Could you could call this option `barRadius`?

Also, I would use the word "rectangle" instead of "rect" in a documentation comment.